src/ripple/nodestore/impl/DatabaseShardImp.cpp

```diff
@@ -1350,12 +1484,48 @@ DatabaseShardImp::getCache(std::uint32_t seq)
     return std::make_pair(pCache, nCache);
 }
 
-std::uint64_t
-DatabaseShardImp::available() const
+bool
+DatabaseShardImp::sufficientStorage(
+    std::uint32_t numShards,
+    bool isHistorical,
+    std::lock_guard<std::mutex> const&) const
 {
     try
     {
-        return boost::filesystem::space(dir_).available;
+        std::vector<std::uint64_t> capacities;
+
+        // Get the available storage for each historical path
```

Comment:
Nit: We usually put comment pertaining to the block inside the block, not before the block (for the `else` block as well)

Reply:
Done